backend/letter/admin.py

```diff
@@ -52,7 +52,8 @@ class PreservedLetterRoleAdmin(admin.StackedInline):
 @admin.register(models.PreservedLetter)
 class PreservedLetterAdmin(admin.ModelAdmin):
     list_display = ["name", "description"]
-    fields = ["name", "description"]
+    fields = ["name", "description", "contributors"]
```

Comment:
Instead of these fields, you can also use the existing core fieldsets (just as you did for `HistoricalPerson`). Not a hard requirement, just pointing it out for consistency's sake.

```py
    fieldsets = [
        core_admin.named_fieldset,
        core_admin.contributions_fieldset,
    ]
```